packages/cli/src/error_handler.ts

```diff
@@ -134,3 +156,10 @@ const errorHasCauseMessage = (
     typeof error.cause.message === 'string'
   );
 };
+
+/**
+ * If the parser finished processing arguments, attempt extracting subcommand information.
+ */
+export const extractSubCommands = (yargs: Argv): string => {
+  return yargs.parsed ? yargs.parsed.argv._.join(' ') : '';
+};
```

Comment:
This should be pulled out of here to either `cli-core` or in it's own file in `backend-cli`